README.md

```diff
@@ -11,30 +12,30 @@ Then, run the following command to install the dependencies:
 make install
 ```
 
-### Run linter
+## Run linter
 ```bash
 make lint
 ```
 
-### Run prettier
+## Run prettier
 ```bash
 make prettier
 ```
 
-### Run all tests
+## Run all tests
 ```bash
 make test
 ```
 
-### Run specific tests
-A regular expression can be used to only run specific tests.
+## Run tests with extra options
+`forge test` can be used directly to provide extra options, like running only specific tests.
 
 ```bash
-forge test -m <REGEX>
-forge test -m testName
-forge test -m ':ContractName\.'
+forge test --mt testName
```

Comment:
Just learned about match test instead of match. What is the difference?

Reply:
There's none. `-m` is a legacy flag from the DappTools era, and it's deprecated in favor of `--match-test` or `--mt`. This is because Foundry has new flags for matching based on contract name and path name, plain `-m` was too vague.